docs/Concepts/Privacy/Privacy-Overview.md

```diff
@@ -4,11 +4,11 @@ description: Privacy
 # Privacy
 
 Privacy in Pantheon refers to the ability to keep transactions private between the involved parties. 
-Other parties cannot access the transaction content, sending party, or list of participating parties. 
+Other parties cannot access the transaction content, sending party, or list of participating parties.
```

Comment:
@MadelineMurray Is there a consequence for using the term "sending" here? Additionally, in the latest revision, I added your changes plus I substituted parties for participants, just to stay consistent with the introductory paragraph. If there's a nuance to these two terms that I'm missing, let me know.

Reply:
As discussed, I'd use 'participants' over 'parties' for consistency.  And as participants implies the sender this could be reworded as: 
Other parties cannot access the transaction content or list of participants.